activiti-spring-process-extensions/pom.xml

```diff
@@ -24,6 +24,10 @@
             <groupId>org.activiti</groupId>
             <artifactId>activiti-spring</artifactId>
         </dependency>
+        <dependency>
```

Comment:
I don't like adding this dependency here. Would be great if [VariableDefinition](https://github.com/Activiti/activiti-core-common/blob/develop/activiti-connector-model/src/main/java/org/activiti/core/common/model/connector/VariableDefinition.java) was moved to a common module since it's not related to connectors only.

Reply:
Totally agree @balsarori. I'll find a better place for it.